benchmarks/experiment_runner.py

```diff
@@ -285,6 +286,41 @@ def collect_profile_to_metrics(self, prof, metrics):
     metrics[
         "per_iter_cuda_time_s"] = total_cuda_time / self._args.iterations_per_run
 
+  def get_xla_cpu_fallback_ops(self, met):
+    return set(name for name in met.counter_names() if self.is_aten_op(name))
```

Comment:
Are there duplicates in this list? If so, what does it mean and should we collect that too?